### DISCUSSION TITLE
Edit screen refresh multiple times when switching

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1240] [Q&A] [answered]
When I add a Mandeeps module to a page I get several page loads when I open the Vanjaro bar. See screen cast.
http://somup.com/crhIfXFMhi.

Its not the capacity of my machine
![image](https://user-images.githubusercontent.com/6786299/118076662-3f0a0200-b378-11eb-875a-c751843a029a.png)

**Comment by meetmandeep** [comment, marked as answer]
Thanks for reporting this. This has been resolved in one of the nightly builds this week. The experience has been improved for final release.
